docs/en/observability/configure-elastic-serverless-forwarder.asciidoc

```diff
@@ -0,0 +1,278 @@
+:aws: AWS
+
+[[configure-elastic-serverless-forwarder]]
+= Configure Elastic Serverless Forwarder
+
+++++
+<titleabbrev>Configure</titleabbrev>
+++++
+:keywords: serverless
+:description: Configure the Elastic Serverless Forwarder to collect logs from Amazon Web Services ({aws}) and send the data to Elastic.
+
+// add overview context i.e. what is created on deployment, what needs to be done next, contextualise and link to sample config
+
+[[lambda-permissions-policies]]
+== Permissions and policies
+
+A Lambda function's execution role is an {aws} Identity and Access Management (IAM) role that grants the function permission to access {aws} services and resources. This role is automatically created when the function is deployed and Lambda assumes this role when the function is invoked.
+
+When you provide the ARNs of {aws} resources the forwarder will interact with, the Cloudformation template will create the correct IAM role with the appropriate IAM policies.
+
+You can view the execution role associated with your function from the **Configuration -> Permissions** section. By default this role starts with the name **serverlessrepo-elastic-se-ElasticServerlessForward-**. When the role is created, a custom policy is added to grant Lambda minimum permissions to be able to use the configured SQS queue, S3 buckets, Kinesis data stream, CloudWatch Logs log groups, Secrets manager (if using), and SQS replay queue.
+
+The Lambda function is granted the following `ManagedPolicyArns` permissions. These are automatically added by default if relevant to the Events configuration:
+
+[source, bash]
+----
+arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+arn:aws:iam::aws:policy/service-role/AWSLambdaKinesisExecutionRole
+arn:aws:iam::aws:policy/service-role/AWSLambdaSQSQueueExecutionRole
+----
+
+In addition to these basic permissions, the following permissions are added when the function is created by the Cloudformation template of the function:
+
+* For SQS queue resources specified in the `SQS_CONTINUE_URL` and `SQS_REPLAY_URL` environment variables, the following action is allowed:
+
+  `sqs:SendMessage`
+
+* For S3 bucket resources specified in the `S3_CONFIG_FILE` environment variable, the following action is allowed on the S3 buckets' config file object key:
+
+  `s3:GetObject`
+
+* For every S3 bucket resource that SQS queues are receiving notifications from, the following action is allowed on the S3 buckets:
+
+  `s3:ListBucket`
+
+* For every S3 bucket resource that SQS queues are receiving notifications from, the following action is allowed on the S3 buckets' keys:
+
+  `s3:GetObject`
+
+* For every Secret Manager secret that you want to refer in the yaml configuration file, the following action is allowed
```

Comment:
```suggestion
* For every Secret Manager secret that you want to refer in the yaml configuration file, the following action is allowed:
```

I'd also choose one style and stick with it across files:

* `config.yaml` file
* config YAML file
* YAML configuration file